translator/translate/otel/common/common.go

```diff
@@ -32,6 +33,18 @@ const (
 	XrayKey                            = "xray"
 	OtlpKey                            = "otlp"
 	JmxKey                             = "jmx"
+	ActiveMqKey                        = "activemq"
+	CassandraKey                       = "cassandra"
+	HbaseKey                           = "hbase"
+	HadoopKey                          = "hadoop"
+	JettyKey                           = "jetty"
+	JvmKey                             = "jvm"
+	KafkaKey                           = "kafka"
+	KafkaConsumerKey                   = "kafka-consumer"
+	KafkaProducerKey                   = "kafka-producer"
+	SolrKey                            = "solr"
+	TomcatKey                          = "tomcat"
+	WildflyKey                         = "wildfly"
```

Comment:
nit: These seem very JMX specific. I don't think they need to be in common.

Reply:
Moved to the processor's translator